docs/src/_getting-started/meltano-at-a-glance.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Meltano at a Glance
+description: If you want to know why you should use Meltano, here's a quick overview of the basics of all of it.
+layout: doc
+weight: 1
+---
+
+This document is intended to give you enough technical understanding of Meltano to becoming excited about it and wanting to use it! It's not going to teach you how to use it, we got Tutorials & How Tos for that. When you're ready to start your first Meltano project, we recommend you dive right into our Tutorial.
+
+### What users say
+
+_"For us it's a better day at work when we can use Meltano."_ - Nino Müller, Head of Technology at Substring
+
+
+_"I love Meltano because it’s so pleasant to use with its DevOps and Everything-as-Code style. It is easy to set up, flexible, and integrates with pretty much any orchestrator as well as the data built tool (dbt)"._  - Martin Morset
```

Comment:
```suggestion
_"I love Meltano because it’s so pleasant to use with its DevOps and Everything-as-Code style. It is easy to set up, flexible, and integrates with pretty much any orchestrator as well as dbt (data build tool)"._  - Martin Morset
```